examples/lock-app/nrfconnect/main/AppTask.cpp

```diff
@@ -385,6 +390,12 @@ void AppTask::ActionCompleted(BoltLockManager::Action_t aAction)
         LOG_INF("Unlock Action has been completed");
         sLockLED.Set(false);
     }
+
+    if (sAppTask.mButtonTriggeredActon)
```

Comment:
Is there a sane way to have the caller pass the actor here, like they did for `ActionInitiated`?

Reply:
I answered on this remark in the comment above.